requirements.txt

```diff
@@ -1,5 +1,9 @@
+opencv-python==4.1.1.26
```

Comment:
I just verified and the functions used from OpenCV are unchanged since version 3. The 4th release is quite a hefty one (noticed when I installed the branch as it was downloading the new release), so I would suggest putting
`opencv-python>=3.4.5.20` instead (which is the most common stable release I've seen from the package).

Then the user will be free to use whichever compatible version from that one on

Reply:
Indeed, OpenCV is a hefty lib, but its size does not seem to change much between version 3 and 4. (Even the version you're suggesting seems heavier):

![Screenshot from 2020-04-20 09-38-29](https://user-images.githubusercontent.com/6331909/79726282-d5413500-82ea-11ea-94d4-8421e361bfb7.png)

More over, it's always better to have more recent versions as you suggested when reviewing other new dependencies. 
Besides, bug fixes and enhancements of version 3 [have been propagated to version 4](https://github.com/opencv/opencv/wiki/ChangeLog#version3410)
So, I think it would be better to lower bound with the `4.3.0`

Reply:
My bad for the size, I must have mistaken this with a previous version!

However, regarding backward compatibility, I disagree as there might be fixes and improvements in later versions, but regarding the functions being used in this PR, I don't see any behaviour change.

In many projects, backward compatibility is a struggle, but here with the limited dependencies in OpenCV (stable for many versions), I cannot see why forbidding the user to link the dependency to his version, if compatible? If at some point, there is a behaviour difference, we can easily open an issue and just raise the lower bound of version requirement, don't you think?

Reply:
Okay, let's do it